### ISSUE TITLE
Allow Sorting In Playlists

### ISSUE THREAD

**Issue by highschooldropout** [martpie/museeks#645]
Currently, there is no way to sort by Title, Duration, Artist, Album or Genre in Playlists. Hopefully this can be added.

**Comment by martpie** [comment]
Yes, this will be tackled in the UI rewrite I'm working on :) no ETA tough

**Comment by martpie** [comment]
This is unrelated to this particular issue, please see https://github.com/martpie/museeks/issues/130
